windows/code/RNDeviceInfoCPP.h

```diff
@@ -0,0 +1,442 @@
+#pragma once
```

Comment:
Am I strange for thinking a .h file should be a header file and the .cpp would normally contain the implementation 🤔  ?

Reply:
Yes, that'd be accurate for classes you'd like to include in order to keep the API in the header, without code.
In this case it can be either way, since the API won't be consumed directly by other C++ code, but rather compiled as a module.
I've chosen to follow this sample when choosing this: https://github.com/microsoft/react-native-windows-samples/blob/master/samples/NativeModuleSample/cppwinrt/windows/NativeModuleSample/DataMarshallingExamples.h

Reply:
I can divide it into a .h and .cpp if you prefer.

Reply:
I don't feel that strongly about it - I feel documentation is frequently harder to do well than code, and a coordinated change across both is very difficult, so if there is existing documentation in the form of an example then I would lean towards conforming with it - TL;DR: - LGTM ;-)